euth_wagtail/assets/scss/components/_a4-comments.scss

```diff
@@ -0,0 +1,390 @@
+.a4-comments__commentbox__form,
```

Comment:
Most of this file was removed in the previous commit (d36ced41ae9bcf407182765185f2c922efcacd0a), could that be? :)

Reply:
it was renamed

Reply:
Actually the file was deleted in d36ced41ae9bcf407182765185f2c922efcacd0a and readded (with changes) in 845a99da1cbbd13a83d49ae0d166c67a9323e16c - just looks a bit funny when you go through the PR commit by commit. Anyhow, never mind :)